### ISSUE TITLE
Parsing error: Cannot find module '@babel/plugin-syntax-decorators'

### ISSUE THREAD

**Issue by nickautomatic** [preactjs/eslint-config-preact#27]
When I install `eslint-config-preact` in an existing project, running ESLint results in the following error:

```
0:0  error  Parsing error: Cannot find module '@babel/plugin-syntax-decorators'
Require stack:
- /app/node_modules/eslint-config-preact/node_modules/@babel/core/lib/config/files/plugins.js
- /app/node_modules/eslint-config-preact/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/eslint-config-preact/node_modules/@babel/core/lib/index.js
- /app/node_modules/eslint-config-preact/node_modules/@babel/eslint-parser/lib/worker/babel-core.cjs
- /app/node_modules/eslint-config-preact/node_modules/@babel/eslint-parser/lib/worker/ast-info.cjs
- /app/node_modules/eslint-config-preact/node_modules/@babel/eslint-parser/lib/worker/handle-message.cjs
- /app/node_modules/eslint-config-preact/node_modules/@babel/eslint-parser/lib/client.cjs
- /app/node_modules/eslint-config-preact/node_modules/@babel/eslint-parser/lib/index.cjs
- /app/node_modules/@eslint/eslintrc/dist/eslintrc.cjs

Make sure that all the Babel plugins and presets you are using
are defined as dependencies or devDependencies in your package.json
file. It's possible that the missing plugin is loaded by a preset
you are using that forgot to add the plugin to its dependencies: you
can workaround this problem by explicitly adding the missing package
to your top-level package.json
```

If I add `@babel/plugin-syntax-decorators` as an explicit dependency then this issue goes away. However, it looks like [the most recent PR](https://github.com/preactjs/eslint-config-preact/pull/24) in this repo may have already addressed this issue by removing the dependency on that package. Although that's been merged to `master`, it was merged after the most recent release (1.3.0)

Just wondering if there's any plan to publish a new release which includes that fix at all? Or failing that, is adding `@babel/plugin-syntax-decorators` as a dependency the best way to get round this issue for now?

Thanks!

**Comment by rschristian** [comment]
Sorry, none of us were watching this repo (and I don't think we really use ESLint ourselves).

Released as `1.4.0`

**Comment by nickautomatic** [comment]
Fantastic, thanks @rschristian!👍
